### ISSUE TITLE
Vizy ignores matrix validation rules

### ISSUE THREAD

**Issue by dyerc** [verbb/vizy#48]
**Description**
If a matrix field within a Vizy block contains required fields or has a minimum / maximum number of blocks these don't seem to be checked by Vizy when saving or publishing the page.

**Steps to reproduce**
1. Create a matrix field with a required field (or minimum number of blocks)
2. Add the matrix field to a Vizy block
3. Attempt to save an entry with the matrix field left blank

**Additional info**
- Plugin version: 1.0.2
- Craft version: 3.6.17
- Multi-site: No

**Comment by xhuang9** [comment]
I have experienced the same issue, by just adding one layer of matrix within Vizy block.

- When there is a minimum field requirement on the matrix block. On save entry, Vizy will creating matrix block contain empty fields to fill up and match the minimum requirement and save it regardless, instead of stop the saving and pop an error message. 
- You can save an entry contain matrix block with empty require fields wrapped with a Vizy field.

Plugin version 1.0.2
Craft version: 3.6.6
multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.5](https://github.com/verbb/vizy/releases/tag/1.0.5)
